beta-src/src/state/game/game-api-slice.ts

```diff
@@ -708,6 +727,12 @@ const gameApiSlice = createSlice({
             }
           });
         }
+      })
+      // Fetch Game Messages
+      .addCase(fetchGameMessages.fulfilled, (state, action) => {
+        if (action.payload) {
+          state.messages = action.payload;
```

Comment:
Question: isn't the payload here going to contain meta data like `msg`, `success`, etc? If so don't we need to assign the `data`, not the entire payload.